cluster/gce/upgrade.sh

```diff
@@ -374,14 +387,15 @@ function do-node-upgrade() {
   # Do the actual upgrade.
   # NOTE(zmerlynn): If you are changing this gcloud command, update
   #                 test/e2e/cluster_upgrade.go to match this EXACTLY.
-  local template_name=$(get-template-name-from-version ${SANITIZED_VERSION} ${NODE_INSTANCE_PREFIX})
+  local template_name
+  template_name=$(get-template-name-from-version "${SANITIZED_VERSION}" "${NODE_INSTANCE_PREFIX}")
   local old_templates=()
-  local updates=()
-  for group in ${INSTANCE_GROUPS[@]}; do
-    old_templates+=($(gcloud compute instance-groups managed list \
+  for group in "${INSTANCE_GROUPS[@]}"; do
```

Comment:
are you _sure_ this works exactly the same as before? I think we actually want globbing/splitting here

Reply:
No, they are not strictly equal. But I do not think globbing of individual elements is necessary here. refer to https://github.com/koalaman/shellcheck/wiki/SC2068